activesupport/lib/active_support/core_ext/digest/uuid.rb

```diff
@@ -16,13 +16,15 @@ module UUID
     #
     # See RFC 4122 for details of UUID at: https://www.ietf.org/rfc/rfc4122.txt
     def self.uuid_from_hash(hash_class, uuid_namespace, name)
+      # rubocop:disable Style/CaseLikeIf
```

Comment:
I think we want to avoid pragmas like these.  I do see [two](https://github.com/rails/rails/blob/722ca6ba353cf44e3b66d77a6e174a2715c8cd5a/railties/lib/rails/all.rb#L3) [places](https://github.com/rails/rails/blob/722ca6ba353cf44e3b66d77a6e174a2715c8cd5a/railties/test/commands/routes_test.rb#L213) in the code base where we currently use them, but I think those are special exceptions.